extensions/standard-processors/processors/GetFile.cpp

```diff
@@ -228,42 +228,48 @@ void GetFile::pollListing(std::queue<std::string> &list, const GetFileRequest &r
 bool GetFile::acceptFile(std::string fullName, std::string name, const GetFileRequest &request) {
   logger_->log_trace("Checking file: %s", fullName);
 
+#ifdef WIN32
+  struct _stat64 statbuf;
+  if (_stat64(fullName.c_str(), &statbuf) != 0) {
+    return false;
+  }
+#else
   struct stat statbuf;
+  if (stat(fullName.c_str(), &statbuf) != 0) {
+    return false;
+  }
+#endif
```

Comment:
we could use `utils::file::exists` and then call `utils::file::file_size` and `utils::file::last_write_time` separately, my concern with this, is that we took one (supposedly) atomic `stat`-call and made three separate calls, on the other side, that would limit the number of places `stat` is explicitly called from, moreover if we trace where this `acceptFile` is called from (`file::list_dir`) we could even ignore the possibility of a non-existing file

1. call `stat` explicitly (current)
2. use 3 calls (exists, file_size, last_write_time)
3. use 2 calls (file_size, last_write_time) because we don't care about non-existing files because a) this method is called with existing files b) we will handle them anyway downstream when we actually read it
4. something else (make file_size, last_write_time return optionals, etc )

Reply:
I'd stay with 1